README.md

```diff
@@ -22,20 +22,9 @@ Run the following commands:
 
 ```bash
 git clone https://github.com/nllz/bigbang.git
-conda create -n bigbang python
 cd bigbang
-bash conda-setup.sh
-```
-
-### pip installation
-
-Run the following commands:
-
-```bash
-git clone https://github.com/sbenthall/bigbang.git
-# optionally create a new virtualenv here
-pip install -r requirements.txt
-python setup.py develop
+conda create -f bigbang environment.yaml
```

Comment:
From documentation, it seems like `conda env create -f environment.yaml` also works. Is there any reason we need to specify the env name in this command?